lib/web-test-proc-macro/src/lib.rs

```diff
@@ -0,0 +1,55 @@
+extern crate proc_macro;
+
+use proc_macro::TokenStream;
+use syn::*;
+use quote::quote;
+
+#[proc_macro_attribute]
+pub fn web_test(_args: TokenStream, input: TokenStream) -> TokenStream {
```

Comment:
What this macro is doing? I'm not good in reading macros definitions, a short comment would help here

Reply:
I added this doc
```rust
/// #[web_test] creates a [320.0, 240.0] div with id = fn_name and appends it
/// into the document.
```

and this note

```rust
            // We insert Container::new("Tests", fn_name, 320.0, 240.0)
            // at the beginning of the function block.
            parsed.block.stmts.insert(0, stmt);
```

Reply:
The docs shold mention how to use it. It's actially not obvious to me reading the current docs and signature.

Reply:
We now have an usage example for `#[web_test]` and `#[web_test]`